### PR TITLE
feat(Quality): starting quality file on basic objects

### PR DESCRIPTION
**Pull request by BotellaA**

### CODE REVIEW

File: src/geode/geometry/quality.cpp [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        const auto edge_bc_l2 = edge_bc.length2();
+        const auto edge_ac_l2 = edge_ac.length2();
+        const auto edge_ad_l2 = edge_ad.length2();
+        const auto edge_bd_l2 = edge_bd.length2();
+        const auto edge_cd_l2 = edge_cd.length2();
+        const auto A = edge_ab_l2 > edge_bc_l2 ? edge_ab_l2 : edge_bc_l2;
+        const auto B = edge_ac_l2 > edge_ad_l2 ? edge_ac_l2 : edge_ad_l2;
+        const auto C = edge_bd_l2 > edge_cd_l2 ? edge_bd_l2 : edge_cd_l2;
+        const auto D = A > B ? A : B;
+        const auto longuest_edge = D > C ? std::sqrt( D ) : std::sqrt( C );
```
Comment (panquez) [inline src/geode/geometry/quality.cpp:60]:
```suggestion
        const auto longest_edge_length = D > C ? std::sqrt( D ) : std::sqrt( C );
```

File: src/geode/geometry/quality.cpp [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        const auto A_area2 = edge_ab.cross( edge_bc ).length();
+        const auto B_area2 = edge_ab.cross( edge_ad ).length();
+        const auto C_area2 = edge_ac.cross( edge_ad ).length();
+        const auto D_area2 = edge_bc.cross( edge_cd ).length();
+        const auto constant = std::sqrt( 6 ) / 12.;
```
Comment (panquez) [inline src/geode/geometry/quality.cpp:65]: constexpr?

**Review by panquez — COMMENTED** [review]
Could you add a test with a minima an isotropic tet, a sliver, and one in between these two cases?

File: src/geode/geometry/quality.cpp (hunk 2 above)
Comment (BotellaA) [reply src/geode/geometry/quality.cpp:65]: sqrt is not always constexpr...

**Review by panquez — APPROVED** [review]

**Review by francoisbonneau — APPROVED** [review]

**Pull request merged**